src/CMakeLists.txt

```diff
@@ -10,10 +10,12 @@ endif()
 add_subdirectory(CubicInterpolation)
 add_subdirectory(detail)
 
-target_link_libraries(CubicInterpolation
-    CONAN_PKG::eigen
-    CONAN_PKG::boost
-    )
+target_link_libraries(CubicInterpolation PUBLIC
+    Eigen3::Eigen
+    Boost::boost
+)
+get_target_property(INCS Eigen3::Eigen INTERFACE_INCLUDE_DIRECTORIES)
+message(STATUS "EIGEN INCLUDE ${INCS}")
```

Comment:
Is there any deeper reason for this output, or a debug artifact?

Reply:
debug artifact